recipes/dep_cats_permits/create.sql

```diff
@@ -1,3 +1,26 @@
+/*
+DESCRIPTION:
+    1. Import geocoded dep_cats_permits to EDM database using PSTDIN
+    2. Filter based on ID and request type
+        - exclude cancelled
+        - exclude G permits
+        - exclude C permits unless they are non-registrations
+        - exclude CA permits unless they are not work permits and not expired
```

Comment:
Does this logic make sense?

Reply:
this logic was written based on a email from EARD, unfortunately it's not documented anywhere else

Reply:
@AmandaDoyle I know EARD requested C permit research files. Do we know whether we are excluding all C-permits? Does the logic look correct to you?

Reply:
I'd compare the outputs to make sure that it returns the same records.  Looking at past issues (https://github.com/NYCPlanning/ceqr-app-data-archive/issues/79 and https://github.com/NYCPlanning/ceqr-app-data-archive/issues/80) 
There should be an output for research **Air: DEP data subset for research #80**
All C-permits should be excluded from final output.  We can reconfirm with EARD